browser/resources/settings/brave_system_page/brave_vpn_page.ts

```diff
@@ -66,7 +66,9 @@ export class SettingsBraveVpnPageElement
     this.initialProtocolValue_ = this.getCurrentPrefValue()
     this.updateState()
     this.addWebUiListener('brave-vpn-state-change', this.onVpnStateChange.bind(this))
+    // <if expr="is_win">
```

Comment:
Can it be better to use `enable_vpn_wirgard` so that you don't have to add platforms in the future?

Or we might be able to remove this as this file is compiled only when `enable_vpn_wirgard` is true.

Reply:
This code should work only for win when enable_brave_vpn_wireguard enabled, highly likely I will refactor it later when OSX part will be ready